File: Challenges/Coding World Cup/2023/Day 33 - BAN vs SL - Decode Strategy.java
public class Solution {
    static String decodeString(String s) {
        String ans = "";
        int n = s.length();

        for (int i = 0; i < n; i++) {
            if (s.charAt(i) == 'z') {
                ans += 'a';
            } else {
                ans += (char) (s.charAt(i) + 1);
            }
        }

        return ans;
    }
}

File: Challenges/Coding World Cup/2023/Day 33 - BAN vs SL - Decode Strategy.cpp
string decodeString(string &s)
{
    // Write your code here.
    string ans = "";
    int n = s.size();

    for (int i = 0; i < n; i++)
    {
        if (s[i] == 'z')
            ans += 'a';
        else
            ans += static_cast<char>((static_cast<int>(s[i]) + 1));
    }

    return ans;
}

